README.md

```diff
@@ -4,7 +4,8 @@
 
```

Comment:
Please create the 3.x branch update the NOTE above.

Reply:
s/3.x/2.x

Reply:
good catch